<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionControlsWidgetClass</class>
 <widget class="QWidget" name="SessionControlsWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>117</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SessionControlsWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="buttonOpenCloseDevices">
     <property name="text">
      <string>Open Devices</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonStartStopDevices">
     <property name="text">
      <string>Start Devices</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonRestartDevices">
     <property name="text">
      <string>Restart Devices</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonRecord">
     <property name="text">
      <string>Start Recording</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonProcessing">
     <property name="text">
      <string>Start Processing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonProperties">
     <property name="text">
      <string>Properties</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
